package.json

```diff
@@ -31,6 +31,7 @@
 		"lint-staged": "^7.0.0",
 		"lolex": "^2.1.3",
 		"prettier": "1.11.1",
+		"sinon": "^6.0.1",
```

Comment:
We have Jest which can do this already

Reply:
Didn't know about that, sorry. I can make another PR to fix that if you wish.

Reply:
That'd be awesome!

Reply:
I tried a bit, but I have to admit I'm not very confident with Jest. You'll find below what I tried, but without luck. As you seem to be a big contributor to Jest, I hope you can point me in the right direction :)

**Attempt 1**

`package.json`
```json
"jest": {
  "testEnvironment": "node",
  "testEnvironmentOptions": {
    "process": {
      "platform": "linux" 
    }
  },
  "testRegex": ".*Test\\.js$"
},
```

**Attempt 2**

`processOpenFileDescriptorsTest.js`
```js
jest.mock('process', () => {
  return {
    platform: 'linux'
  };
});
```

Reply:
Since all test files run in isolation in Jest, you can just assign.

```js
process.platform = 'linux';
```